api/models/risk_of_bias.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const bookshelf = require('../../config').bookshelf;
+const BaseModel = require('./base');
+require('./source');
+require('./trial');
+require('./risk_of_bias_criteria')
+require('../helpers');
```

Comment:
To keep it consistent with the rest of the code, move the `require('./MODEL')` to before the other requires, and remove the unused `require('../helpers')` (see https://github.com/opentrials/api/blob/master/api/models/trial.js for an example of what I mean)